src/pages/billing/Project.js

```diff
@@ -564,26 +564,28 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const tabToTable = {
     workspaces: h(Fragment, [
-      h(WorkspaceCardHeaders, {
-        needsStatusColumn: billingAccountsOutOfDate,
-        sort: workspaceSort,
-        onSort: setWorkspaceSort
-      }),
-      div({ role: 'list', 'aria-label': `workspaces in billing project ${billingProject.projectName}`, style: { flexGrow: 1, width: '100%' } }, [
-        _.flow(
-          _.orderBy([workspaceSort.field], [workspaceSort.direction]),
-          _.map(workspace => {
-            const isExpanded = expandedWorkspaceName === workspace.name
-            return h(WorkspaceCard, {
-              workspace: { ...workspace, billingAccountDisplayName: billingAccounts[workspace.billingAccount]?.displayName },
-              billingProject,
-              billingAccountStatus: billingAccountsOutOfDate && getBillingAccountStatus(workspace),
-              key: workspace.workspaceId,
-              isExpanded,
-              onExpand: () => setExpandedWorkspaceName(isExpanded ? undefined : workspace.name)
+      div({ role: 'table', 'aria-label': `workspaces in billing project ${billingProject.projectName}` }, [
```

Comment:
Note role table now instead of list. Perhaps the list role predated when sortable headers were added.